ferretdb/ferretdb.go

```diff
@@ -67,6 +67,12 @@ type Config struct {
 
 	// Defaults to [io.Discard], effectively disabling logging.
 	LogOutput io.Writer
+
+	// EnabledTelemetry set the telemetry enable or not.
+	// If empty, the telemetry is enabled by default.
```

Comment:
That's not correct

Reply:
I will fix it and update document in the next commit :)

> That's not correct